terminus/generators/opendrive_id_mapper.py

```diff
@@ -0,0 +1,11 @@
+from city_visitor import CityVisitor
+
+
+class OpenDriveIdMapper(CityVisitor):
```

Comment:
@basicNew I think it's better to keep them in another class since we will need to track them to create junctions. So we can make something similar to RNDF generator. Just to keep the interface.